### PR TITLE
arDominionB5Plugin: style report and import pages

### PR DESCRIPTION
**Pull request by sevein**

### CODE REVIEW

File: plugins/arDominionB5Plugin/modules/informationobject/templates/boxLabelCriteria.php [hunk 1]
```diff
@@ -5,27 +5,46 @@
...
-  <h1><?php echo __('%1 - report criteria', ['%1' => $type]); ?></h1>
-  <h2><?php echo render_title($resource); ?></h2>
+  <div class="multiline-header d-flex flex-column mb-3">
+    <h1 class="mb-0" aria-describedby="heading-label">
+      <?php echo __('%1 - report criteria', ['%1' => $type]); ?>
+    </h1>
+    <span class="small" id="heading-label">
+      <?php echo render_title($resource); ?>
+    </span>
+  </div>
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/informationobject/templates/boxLabelCriteria.php:15]: Nice!

File: plugins/arDominionB5Plugin/modules/informationobject/templates/boxLabelCriteria.php [hunk 2]
```diff
@@ -5,27 +5,46 @@
...
 
-<?php slot('after-content'); ?>
-  <ul class="actions nav gap-2">
-    <li><input class="btn atom-btn-outline-success" type="submit" value="<?php echo __('Continue'); ?>"></li>
-    <li><?php echo link_to(__('Cancel'), [$resource, 'module' => 'informationobject'], ['class' => 'btn atom-btn-outline-light', 'role' => 'button']); ?></li>
-  </ul>
+    <div class="accordion">
+      <div class="accordion-item">
+        <h2 class="accordion-header" id="report-criteria-heading">
+          <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#report-criteria-collapse" aria-expanded="false" aria-controls="report-criteria-collapse">
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/informationobject/templates/boxLabelCriteria.php]: aria-expanded="true"?

File: plugins/arDominionB5Plugin/modules/object/templates/validateCsvSuccess.php [hunk 3]
```diff
@@ -37,15 +37,15 @@
       </div>
       <div class="accordion-item">
         <h2 class="accordion-header" id="select-heading">
-          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#select-collapse" aria-expanded="false" aria-controls="select-collapse">
+          <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#select-collapse" aria-expanded="false" aria-controls="select-collapse">
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/object/templates/validateCsvSuccess.php]: aria-expanded="true"

File: plugins/arDominionB5Plugin/modules/informationobject/templates/boxLabelCriteria.php (hunk 2 above)
Comment (jraddaoui) [reply plugins/arDominionB5Plugin/modules/informationobject/templates/boxLabelCriteria.php]: And in all the others below that are expanded by default.

File: plugins/arDominionB5Plugin/modules/sfSkosPlugin/templates/importSuccess.php [hunk 4]
```diff
@@ -45,18 +45,24 @@
         </h2>
         <div id="import-collapse" class="accordion-collapse collapse show" aria-labelledby="import-heading">
           <div class="accordion-body">
-            <?php if (isset($resource)) { ?>
+            <?php if (true || isset($resource)) { ?>
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/sfSkosPlugin/templates/importSuccess.php]: Testing ...

File: lib/helper/QubitHelper.php [hunk 5]
```diff
@@ -92,7 +92,9 @@ function render_b5_field($field, $translation = null, $options = [])
     $name = $field->getName();
     $widget = $field->getWidget();
 
-    if (in_array($field->type, ['checkbox', 'radio']) || $widget instanceof sfWidgetFormSelectRadio) {
+    if (in_array($field->type, ['checkbox', 'radio'])
+    || $widget instanceof sfWidgetFormSelectRadio
+    || $widget instanceof sfWidgetFormChoice) {
```
Comment (jraddaoui) [inline lib/helper/QubitHelper.php]: Syntax checked is not very good with this, but it should be ...

```
if (
    in_array($field->type, ['checkbox', 'radio'])
    || $widget instanceof sfWidgetFormSelectRadio
    || $widget instanceof sfWidgetFormChoice
) {
```

**Review by jraddaoui — APPROVED** [review]
Nicely done!

File: plugins/arDominionB5Plugin/modules/sfSkosPlugin/templates/importSuccess.php (hunk 4 above)
Comment (sevein) [reply plugins/arDominionB5Plugin/modules/sfSkosPlugin/templates/importSuccess.php]: :laughing:

File: plugins/arDominionB5Plugin/modules/object/templates/validateCsvSuccess.php (hunk 3 above)
Comment (sevein) [reply plugins/arDominionB5Plugin/modules/object/templates/validateCsvSuccess.php]: Ops, thanks, fixed everywhere.

**Pull request merged**